### PR TITLE
also support input.json

### LINKED ISSUE #13: inputs.json instead of input.json?
Refering to the [tool-specs](https://vforwater.github.io/tool-specs/tool/) it should be `inputs.json`, but json2args is looking for `input.json`.

Two possible ways:
* change `input.json` -> `inputs.json`  here in json2args for Python
* change `inputs.json` -> `input.json` in json2aRgs and tool specs

### PR DESCRIPTION
**Pull request by mmaelicke**
closes #13

### CODE REVIEW

**Comment by mmaelicke** [comment]
@AlexDo1 can you run the tests locally? I can't find the problem, as the tests work just fine on my machine and the error produced in the CI does not make any sense to me.

File: json2args/util.py [hunk 1]
```diff
@@ -4,13 +4,34 @@
 from yaml import load, Loader
 
 
-CONF_FILE = '/src/tool.yml'
-PARAM_FILE = '/in/inputs.json'
+CONF_FILE = ['/src/tool.yml']
+PARAM_FILES = ['/in/input.json', '/in/input.json']
```
Comment (AlexDo1) [inline json2args/util.py]: one should be `inputs.json`

**Comment by AlexDo1** [comment]
I get the same error, there seems to be something wrong with loading the batched array data:
https://github.com/hydrocode-de/json2args/blob/95d7d029b80b1c83acf93c7ae2fcba658456ee24/tests/test_load_data.py#L95-L105
```python
>       assert arr.flatten()[0] == 0
E       assert 40.0 == 0
```

**Comment by AlexDo1** [comment]
I think the problem is that the order of the paths is not guaranteed, so you cannot test for the first value, you also made this comment on the assert in the next line:
https://github.com/hydrocode-de/json2args/blob/95d7d029b80b1c83acf93c7ae2fcba658456ee24/tests/test_load_data.py#L101-L103

This is the order of paths for the batched arrays, in your test you assume that `batch1` is the first array:
```python
[PosixPath('/home/alexd/Projekte/VFORWaTeR/Github/json2args/tests/arrays/batch2/arange.dat'),
 PosixPath('/home/alexd/Projekte/VFORWaTeR/Github/json2args/tests/arrays/batch1/arange.dat'),
 PosixPath('/home/alexd/Projekte/VFORWaTeR/Github/json2args/tests/arrays/batch3/arange.dat')]
```

You could change the test to just check if 0 exists in the array as you did with the other assertion.

You could also sort the paths when loading them to ensure consistent behavior:
https://github.com/hydrocode-de/json2args/blob/95d7d029b80b1c83acf93c7ae2fcba658456ee24/json2args/data.py#L44-L45
-> 
```python 
paths = sorted([Path(v).resolve() for v in glob(value)])```

**Review by AlexDo1 — CHANGES_REQUESTED** [review]

File: json2args/util.py (hunk 1 above)
Comment (mmaelicke) [reply json2args/util.py]: Thanks...

The 32°C in office really demand a tribute

**Comment by mmaelicke** [comment]
Thank you so much, now it works

**Pull request merged**